src/map/packet_system.cpp

```diff
@@ -2631,11 +2631,12 @@ void SmallPacket0x05C(map_session_data_t* session, CCharEntity* PChar, CBasicPac
         PChar->loc.p.x = RBUFF(data, (0x04));
         PChar->loc.p.y = RBUFF(data, (0x08));
         PChar->loc.p.z = RBUFF(data, (0x0C));
+        uint16 extras = RBUFW(data, (0x14));
         PChar->loc.p.rotation = RBUFB(data, (0x1F));
 
         if (RBUFB(data, (0x1E)) != 0)
         {
-            luautils::OnEventUpdate(PChar, EventID, 0);
+            luautils::OnEventUpdate(PChar, EventID, 0, extras);
```

Comment:
https://github.com/DarkstarProject/darkstar/blob/bcnmshit/src/map/packet_system.cpp#L2628

Reply:
Removed onEventUpdate 4-parameter overload in further commit.